src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,17 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
```

Comment:
Consider using a variable for the border color to keep consistent styling throughout the application.